NuRadioReco/detector/filterresponse.py

```diff
@@ -95,44 +92,44 @@ def get_filter_response(frequencies, filter_name):
 if __name__ == "__main__":
     from matplotlib import pyplot as plt
 
-    ff = np.linspace(10 * units.MHz, 500 * units.MHz, 100)
-    df = ff[1] - ff[0]
+    freqs = np.linspace(10 * units.MHz, 500 * units.MHz, 100)
```

Comment:
why is this being changed?

Reply:
ff is also used as a parameter name for some of the functions that are defined in the same file.